packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -0,0 +1,139 @@
+import iam = require('@aws-cdk/aws-iam');
+import lambda = require('@aws-cdk/aws-lambda');
+import { Construct, Duration, Lazy, Stack } from '@aws-cdk/core';
+import { CfnAuthorizer } from '../apigateway.generated';
+import { AuthorizationType, Method } from '../method';
+import { AuthorizerBase, AuthorizerConfig } from './authorizer-base';
+
+/**
+ * Properties for TokenAuthorizer
+ */
+export interface TokenAuthorizerProps {
+
+  /**
+   * An optional name for the authorizer. When provided, this will also be used for the physical id of the
+   * CloudFormation resource of type `AWS::ApiGateway::Authorizer`.
+   *
+   * @default - CDK will use the uniqueId assigned to this construct.
+   */
+  readonly authorizerName?: string;
+
+  /**
+   * The handler for the authorizer lambda function.
+   *
+   * The handler must follow a very specific protocol on the input it receives and the output it needs to produce.
+   * API Gateway has documented the handler's input specification
+   * {@link https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-input.html | here} and output specification
+   * {@link https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-output.html | here}.
+   */
+  readonly handler: lambda.IFunction;
+
+  /**
+   * The request header mapping expression for the bearer token. This is typically passed as part of the header, in which case
+   * this should be `method.request.header.Authorizer` where Authorizer is the header containing the bearer token.
+   * @see https://docs.aws.amazon.com/apigateway/api-reference/link-relation/authorizer-create/#identitySource
+   * @default 'method.request.header.Authorizer'
+   */
+  readonly identitySource?: string;
+
+  /**
+   * The TTL on how long APIGateway should cache the results. Max 1 hour.
+   * Disable caching by setting this to 0.
+   *
+   *  @default - Duration.minutes(5)
```

Comment:
Since this is a specific value, then you can use `@default Duration.minutes(5)` instead of `@default -`. We use `-` when there is no specific value and we just want to describe the default behavior

Reply:
This doesn't look like it was resolved.